docs/reports/mapped_deprecated.md

```diff
@@ -1,13 +1,13 @@
 # Mapped deprecated terms
 | Ontology                                    |   Tot deprecated in Mondo |
 |:--------------------------------------------|--------------------------:|
-| [OMIM](./mapped_deprecated_omim.md)         |                        40 |
-| [DOID](./mapped_deprecated_doid.md)         |                         4 |
-| [NCIT](./mapped_deprecated_ncit.md)         |                         3 |
-| [ORDO](./mapped_deprecated_ordo.md)         |                       113 |
-| [ICD10WHO](./mapped_deprecated_icd10who.md) |                         0 |
-| [ICD10CM](./mapped_deprecated_icd10cm.md)   |                         0 |
+| [ORDO](./mapped_deprecated_ordo.md)         |                       574 |
+| [OMIM](./mapped_deprecated_omim.md)         |                        44 |
+| [NCIT](./mapped_deprecated_ncit.md)         |                        33 |
+| [DOID](./mapped_deprecated_doid.md)         |                         0 |
```

Comment:
@matentzn As you expected in #331, these DO terms now no longer show up here after this fix, which is good.